synapse/lib/scrape.py

```diff
@@ -214,12 +214,25 @@ def windows_path_check(match: regex.Match):
 )
 '''
 
+# https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/62e862f4-2a51-452e-8eeb-dc4ff5ee33cc
+def unc_path_check(match: regex.Match):
+    mnfo = match.groupdict()
+    valu = mnfo.get('valu')
+
+    try:
+        s_chop.uncpath(valu)
```

Comment:
can we actually return the context from here to skip a subsequent need to call s_chop.uncpath() ?  Should we adjust the callback convention to expect fully normalized values to prevent the duplicate parsing calls?

Reply:
I updated it so `uncnorm` (prev `s_chop.uncpath`) validates and converts from a UNC to a URI. The scrape code returns the URI which then makes it so `inet:url` norm method doesn't need to validate/convert again.